kuizy/index.php

```diff
@@ -10,6 +10,9 @@
   <link rel="stylesheet" href="./kuizy.css">
 </head>
 <body>
+  <?php
+  echo phpinfo();
+  ?>
```

Comment:
`<?php echo  XX ?>` の短縮形が推奨されているので `<?= XX ?>` を利用するように意識してください〜
```suggestion
  <?= phpinfo(); ?>
```